draco/analysis/flagging.py

```diff
@@ -28,9 +29,12 @@
 from scipy.ndimage import median_filter, uniform_filter
 
 from caput import config, weighted_median
+from ch_util import rfi, ephemeris
```

Comment:
Remember, you can't depend on `ch_util` in here. For the ephemeris calls, you can probably get away with converting times to LSA's (using an `Observer` object), and then just masking based on that and the source RA's.

For the calls into `ch_util.rfi`, I think you can just copy/move the SIR implementation into `draco.util.rfi` and probably the same with the frequency mask. @ssiegelx might have some ideas in here.